build.cake

```diff
@@ -1,9 +1,9 @@
 #tool nuget:?package=GitVersion.CommandLine
 #tool nuget:?package=vswhere
-#addin nuget:?package=Cake.Figlet
-#addin nuget:?package=Cake.Incubator&version=1.7.1
-#addin nuget:?package=Cake.Git&version=0.16.1
+#addin nuget:?package=Cake.Figlet&version=1.1.0
+#addin nuget:?package=Cake.Git&version=0.17.0
 #addin nuget:?package=Polly
+#tool nuget:?package=SignClient&version=0.9.1&include=/tools/netcoreapp2.0/SignClient.dll
```

Comment:
I would not do it this way. It's now a .NET Core tool. I would do it more like this: https://github.com/reactiveui/refit/blob/master/build/Sign-Package.ps1#L10

Reply:
Alrighty, will add that to our appveyor.yml